plugins/com.google.cloud.tools.eclipse.appengine.libraries/src/com/google/cloud/tools/eclipse/appengine/libraries/repository/LibraryClasspathContainerResolverService.java

```diff
@@ -312,4 +315,15 @@ public void unsetRepositoryService(ILibraryRepositoryService repositoryService)
       this.repositoryService = null;
     }
   }
+
+  @Reference
+  public void setExtensionRegistry(IExtensionRegistry extensionRegistry) {
+    this.extensionRegistry = extensionRegistry;
+  }
+
+  public void unsetExtensionRegistry(IExtensionRegistry extensionRegistry) {
```

Comment:
I don't think you really need the _unset_ since you're declaring it as a static 1-1 dependency.